src/azul/azulclient.py

```diff
@@ -233,3 +235,16 @@ def delete_all_indices(self):
                     logger.info("Would delete index '%s'", index_name)
                 else:
                     es_client.indices.delete(index=index_name)
+
+    def delete_bundle(self, bundle_uuid, bundle_version):
+        logger.info('Deleting bundle %s.%s', bundle_uuid, bundle_version)
+        notification = {
+            'match': {
+                'bundle_uuid': bundle_uuid,
+                'bundle_version': bundle_version
+            }
+        }
+        response = requests.post(url=self.indexer_url+'delete',
```

Comment:
Spaces around operators, please.

Reply:
Also, could you change the definition of `self.indexer_url` so it doesn't end in a slash? You will have to change all references to it. This should be done in a separate commit, just after the renaming commit. In the end this should read

```suggestion
        response = requests.post(url=self.indexer_url + '/delete',
```